contracts/ActionWithdrawLiquidity.sol

```diff
@@ -165,7 +165,7 @@ contract ActionWithdrawLiquidity is GelatoActionsStandard {
                     ETH,
                     _collateralToken
                 )
-            returns (uint256 returnAmount) {
+            returns (uint256 returnAmount, uint256 decimals) {
```

Comment:
I think we dont have to define the mem var here and simply use `returns (uint256 returnAmount, )`

Reply:
yeah or if that happens to make compiler complain then `returns (uint256 returnAmount, uint256)` (no var name for second return) works